bin/lib.mjs

```diff
@@ -6,6 +6,22 @@ const raw = `
   ***bold***
   1. list 1
   2. list 2
+
+  # TEST INTERPOLATION
```

Comment:
We can write an additional test case for checks like this, for now I put it here not to mess with the snapshot.

Reply:
I agree.. lets write more test cases :+1:

Reply:
https://labforward.atlassian.net/browse/WFE-311 created a ticket for this so we can keep track. You can add more test cases here if you can think of any.